server/models/eventsPhases.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const eventsPhasesSchema = new Schema({
+    _id: String,
+    eventId: String,
+    phase1GrantingStartDate: Date,
+    phase1GrantingEndDate: Date,
+    phase2PurchaseStartDate: Date,
+    phase2PurchaseEndDate: Date,
```

Comment:
@may-ben-arie @mbenarie before we will merge it -  I called it `phase2PurchaseStartDate`, but I'm not sure if that was what we were talking about... Is the second phase is for purchasing?